analyzers/its/Directory.Build.targets

```diff
@@ -5,6 +5,8 @@
     <PropertyGroup>
       <BinariesFolder>$(MSBuildStartupDirectory)\..\packaging\binaries</BinariesFolder>
       <AnalysisOutputPath>$(MSBuildStartupDirectory)\output\$(PROJECT)\$(AssemblyName)</AnalysisOutputPath>
+      <DiagnosticsPath>$(MSBuildStartupDirectory)\output\$(PROJECT)\$(PROJECT)-$(AssemblyName)</DiagnosticsPath>
```

Comment:
This generally seems like an over-engineered approach. What we're trying to get is the `Issues.json` file. And that one should generally live in `$(SonarProjectOutputPath)\Issues.json` that we compute below, and that already solves the TFM issue.

This seems to load the file with *.json recursive pattern, so modifying the location should not break it.

Let's just delete these `DiagnosticPath*` variables and use `$(SonarProjectOutputPath)` as the base path for the Issues.json file. The file is in unique directory, so it can have simple hardcoded name, same as the one produced during S4NET run.

Reply:
The problem with using `SonarProjectOutputPath` is that it is a variable path, it can be one of these:

- `Project\Assembly\`
- `Project\Assembly\TFM`

And in the end of the execution, we need to copy them from `output` to `actual` in the format:

```
Project/RuleId-Assembly-TFM?.json
```

In the way it works now, it's pretty simple:
`output/{Project}` contains files called `Assembly-TFM.json`, so we get the project name, we get the filename, slap the ruleId on it and it's done.

If we use the variable-length `SonarProjectOutputPath`, even though it solves the TFM question, we need to deal with the TFM question again to find out if we should look one or two directories up.

I think it's simpler to emit the JSONs at the project directory of `output`, since we can extract project+filename with:

```powershell
$file = [System.IO.FileInfo]$sarifReportPath #{Assembly}-{TFM}.json
$project = ([System.IO.FileInfo]$file.DirectoryName).Name
```